package.json

```diff
@@ -59,7 +59,7 @@
     "mkdirp": "0.5.1",
     "mute-stream": "0.0.5",
     "nativescript-dev-xcode": "0.2.0",
-    "nativescript-doctor": "1.10.0",
+    "nativescript-doctor": "file:../nativescript-doctor/nativescript-doctor-1.10.0.tgz",
```

Comment:
Should be updated with the official version after the `nativescript-doctor` is released :)